example/lib/pages/tile_loading_error_handle.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong/latlong.dart';
+
+import '../widgets/drawer.dart';
+
+class TileLoadingErrorHandle extends StatefulWidget {
+  static const String route = '/tile_loading_error_handle';
+
+  @override
+  _TileLoadingErrorHandleState createState() => _TileLoadingErrorHandleState();
+}
+
+class _TileLoadingErrorHandleState extends State<TileLoadingErrorHandle> {
+  @override
+  Widget build(BuildContext context) {
+    bool _needLoadingError = true;
+
+    return Scaffold(
+      appBar: AppBar(title: Text('Tile Loading Error Handle')),
+      drawer: buildDrawer(context, TileLoadingErrorHandle.route),
+      body: Padding(
+        padding: EdgeInsets.all(8.0),
+        child: Column(
+          children: [
+            Padding(
+              padding: EdgeInsets.only(top: 8.0, bottom: 8.0),
+              child: Text('Turn on Airplane mode and try to move or zoom map'),
+            ),
+            Flexible(
+              child: Builder(builder: (BuildContext context) {
+                return FlutterMap(
+                  mapController: MapController(),
```

Comment:
Sorry for nitpicking - this is a bad example use MapController use. A map controller is not necessary.